integration-test/src/main/resources/testsuites/e2e/aws-e2e-tests-for-native.yaml

```diff
@@ -4,5 +4,6 @@ tests:
     excludedGroups: [ azure_singlerg ]
     classes:
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.distrox.DistroXEncryptedVolumeTest
-      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.distrox.DistroXUpgradeTests
+#TODO The upgrade test suite needs to be moved to a separate job which is long running
```

Comment:
Please remove these lines from here and create a new file: `aws-longrunning-e2e-native-tests.yaml` with content:
```
name: "aws-longrunning-e2e-native-tests"
tests:
  - name: "aws_ongrunning_e2e_native_tests"
    classes:
      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.distrox.DistroXUpgradeTests
```